projects/ad738x-st/src/examples/examples_src.mk

```diff
@@ -0,0 +1,22 @@
+ifeq (y,$(strip $(IIO_EXAMPLE)))
+CFLAGS += -DIIO_EXAMPLE=1 -DIIO_SUPPORT -DINTC_DEVICE_ID=1 -DUSE_STANDARD_SPI
```

Comment:
where is USE_STANDARD_SPI used and why is it defined?

even if the result is the same, would move the definition of INTC_DEVICE_ID in parameters.h. also, any reason why 1 was chosen? I think there is only one interrupt vector controller for stm32 and by default in the other projects it has id 0.

Reply:
the ./drivers/adc/ad738x/ad738x.c file uses for offload spi. in the k1/stm32 we use standard spi
so we need to define this flag. 

https://github.com/analogdevicesinc/no-OS/blob/master/drivers/adc/ad738x/ad738x.c#L348

no idea why INTC_DEVICE_ID is 1, this was from the project i copied to start. will test with 0 and change if i dont see any issue.